terms.html

```diff
@@ -62,10 +70,11 @@
   <dt><dfn data-lt="holders|holder's|holders'">holder</dfn></dt>
   <dd>
 A role an <a>entity</a> might perform by possessing one or more
-<a>verifiable credentials</a> and generating <a>presentations</a> from them.
-A holder is usually, but not always, a <a>subject</a> of the <a>verifiable
-credentials</a> they are holding. Holders store their <a>credentials</a> in
-<a>credential repositories</a>.
+<a>verifiable credentials</a> and generating <a>verifiable presentations</a>,
+both from such <a>verifiable credentials</a> and <a>claims</a> it authors itself.
+A holder is usually, but not always, the <a>subject</a> of a <a>claim</a> within
+the <a>verifiable credentials</a> they are holding.
```

Comment:
```suggestion
A holder can be the <a>subject</a> of a <a>claim</a> within
the <a>verifiable credentials</a> they are holding.
```

Reply:
because `a` can imply a limit of one
```suggestion
A holder can be the <a>subject</a> of any number of <a>claims</a> within
the <a>verifiable credentials</a> they are holding.
```

Reply:
Is this an acceptable middle road for the comments of the both of you?

Reply:
I don't understand the referent of `this`. I'm happy with my suggestion above. I'm less happy with the suggestion from @brentzundel; hence my own.